modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-perspective-view.js

```diff
@@ -9,12 +9,12 @@
  * SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and XSK contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-exports.getPerspective = function () {
-    var perspective = {
-        "name": "XS Migration",
-        "link": "../ide-migration/migration-perspective.html",
-        "order": "820",
-        "image": "exchange"
-    };
-    return perspective;
-}
\ No newline at end of file
+package com.sap.xsk.migration.neo.db.hana.connectivity;
+
+public interface HanaConnector  {
+
+  HanaConnectorRes connect(HanaConnectorArgs connectorArgs);
+
+  void disconnect(String connectionId);
+
+}
```

Comment:
I really don't get this change

Reply:
Maybe try to build the module/whole project with:
- `mvn clean install -P content` - to fetch the content from the xsk-ide-migration project in DirigibleLabs
- `mvn clean install` - to update the license headers